i18n/ar.json

```diff
@@ -170,7 +170,6 @@
 	"srf_paramdesc_hashname": "إذا تم توفيرها وإمكانية تمديد جداول التجزئة فسيؤدي هذا إلى إنشاء مزيج بالاسم المحدد (لا يوجد إخراج مرئي بعدئذ)",
 	"srf-printername-graph": "رسم بياني",
 	"srf-paramdesc-graph-relation": "يحدد ما إذا كانت الموضوعات أو خصائص الاسم آباء أم أطفال",
-	"srf-paramdesc-graph-nameprop": "لتعيين الخاصية التي سيتم استخدامها كموضوع بدلا من الموضوع الفعلي: أي اسم الصفحة",
```

Comment:
Try doing one thing per commit please. This could have been in a dedicated PR

Reply:
So should i revert this commit?

Reply:
No, just an FYI